packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -17,13 +17,17 @@
 import { injectable, inject } from 'inversify';
 import { FileSystem, FileStat } from '@theia/filesystem/lib/common';
 import * as path from 'path';
+import { readdir, remove } from 'fs-extra';
 import * as crypto from 'crypto';
 import URI from '@theia/core/lib/common/uri';
 import { isWindows } from '@theia/core';
 import { PluginPaths } from './const';
 import { PluginPathsService } from '../../common/plugin-paths-protocol';
 import { THEIA_EXT, VSCODE_EXT, getTemporaryWorkspaceFileUri } from '@theia/workspace/lib/common';
 
+const MAX_LOG_SESSIONS_TO_KEEP = 10;
```

Comment:
Hello, this hardcoded should be configurable

Reply:
If you can point me to a quick win example how to expose configurable options in Theia. I can have a look (this is my first contribution to theia and I do not know my way around well yet...).

If the configuration is not trivial, I would prefer to keep it out of this PR
and start with alignment to VSCode implementation of keeping 10 **hard-coded**
last sessions.
- https://github.com/Microsoft/vscode/blob/613447d6b3f458ef7fee227e3876303bf5184580/src/vs/code/electron-browser/sharedProcess/contrib/logsDataCleaner.ts#L32

Anyhow in a **Web IDE** scenario we may want a more advanced configuration
assuming disc space on the containers is a somewhat costly resource:
e.g: Max N Folders **AND** max Y logs size in megabytes.
So the configuration option may need to be "attacked" **separately** regardless.

Reply:
hello @bd82, here is an example for 'plugins' which can be configured through cli with 'plugins' arg

Reply:
I think the link is missing,
But is something like this what you are referring to?

- https://github.com/eclipse-theia/theia/blob/6347c5ccbf016b7cfa70976f69616eb1bb49c5b0/packages/plugin-ext/src/main/node/plugin-deployer-impl.ts#L45-L46

Reply:
https://github.com/eclipse-theia/theia/blob/615e817b68e114531282a8a0af34c5af08c573c7/packages/plugin-ext/src/main/node/plugin-cli-contribution.ts

Sorry it was this link yes

Reply:
Done